src-vue/src/plugins/store.ts

```diff
@@ -65,6 +68,9 @@ export const store = createStore<FlightCoreStore>({
             northstar_is_running: false,
             origin_is_running: false,
 
+            player_count: -1,
+            server_count: -1,
```

Comment:
Nah, it should still be a number; maybe 0?

Reply:
In that case I would just stick with `-1` tbh. Cause it indicates more clearly that something went wrong compared to `0` which is valid player/server count. Sure we're unlikely to ever hit that (hopefully) but if some API error causes FlightCore to be unable to load the playercout, the user will see `0 players, 0 servers` and might just think that Northstar is _"dead"_.

Reply:
I don't think users should notice when something goes wrong anyway; FC displaying `-1` will mean to both of us that something went wrong while fetching stats, users will rather think that FC is broken :/
Whatever value you choose, please display a message instead of raw value.

Reply:
Done in f2d4952c9fc98d8bc547d1ade7136ad621207410

Rendered:
![image](https://user-images.githubusercontent.com/40122905/212110066-d113616f-0f8c-4621-8efa-320ca9cdc89d.png)